homework/vector-of-shared-ptrs/vectorFunctions.cpp

```diff
@@ -0,0 +1,35 @@
+
+#include "vectorFunctions.hpp"
+#include <iostream>
+
+std::vector<std::shared_ptr<int>> vector_ptrs;
+
+std::vector<std::shared_ptr<int>> generate(int count){
+
+    for(auto i = 0; i < count; ++i){
+        vector_ptrs.push_back(std::make_shared<int>(i));
+    }
+    return vector_ptrs;
+}
+void print(std::vector<std::shared_ptr<int>> vector_ptrs){
+    for(auto i : vector_ptrs){
+        std::cout << *i << " \n";
+    }
+}
+void add10(std::vector<std::shared_ptr<int>> vector_ptrs){
+    for(auto i : vector_ptrs){
```

Comment:
Add space after `if`, `for`, `while` `function` etc...

Add space between operator like `+` `-` `/` `:` `&&` etc..